config/service_config.go

```diff
@@ -73,6 +73,7 @@ type ServiceConfig struct {
 	ExecuteLimit                string            `yaml:"execute.limit" json:"execute.limit,omitempty" property:"execute.limit"`
 	ExecuteLimitRejectedHandler string            `yaml:"execute.limit.rejected.handler" json:"execute.limit.rejected.handler,omitempty" property:"execute.limit.rejected.handler"`
 	Auth                        string            `yaml:"auth" json:"auth,omitempty" property:"auth"`
+	NotRegister                 bool              `yaml:"register" json:"register,omitempty" property:"register"`
```

Comment:
The field name is `NotRegister`, but in the yaml the name is `register`, is that reasonable?

Reply:
Oh, that's a typo, I would fix it.